package.json

```diff
@@ -5,12 +5,13 @@
   "repository": "https://github.com/angular/ts-minify",
   "license": "Apache-2.0",
   "scripts": {
-    "prepublish": "npm install tsd@^0.6.3 && tsd install node && tsd install mocha && tsd install chai",
+    "prepublish": "npm install tsd@^0.6.3 && tsd install node && tsd install mocha && tsd install chai && tsd install fs-extra",
```

Comment:
have you considered adding a `tsd.json` file to contain all these required typings?